.github/workflows/aws-eks-verify.yml

```diff
@@ -0,0 +1,74 @@
+name: aws eks terraform verification
+
+on:
+  push:
+    paths:
+      - ".github/actions/terraform-*/action.yml"
+      - ".github/workflows/aws-eks-verify.yml"
+      - "aws/eks/eks-terraform-scripts/**"
+      - "!**.md"
+  pull_request:
+    paths:
+      - ".github/actions/terraform-*/action.yml"
+      - ".github/workflows/aws-eks-verify.yml"
+      - "aws/eks/eks-terraform-scripts/**"
+      - "!**.md"
+  workflow_dispatch:
+
+env:
+  TF_IN_AUTOMATION: true
+  TF_INPUT: false
+  TF_CLI_ARGS: "-no-color"
+  WORKING_DIRECTORY: aws/eks/eks-terraform-scripts
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
```

Comment:
Can we please use a specific version? 

This is not a best practice.

Reply:
Fixed to 20.04, thanks